config/initializers/pester.rb

```diff
@@ -1,16 +1,12 @@
-Pester.configure do |c|
-  c.logger = Rails.logger
-
-  c.environments[:schema_refresh] = {
-    delay_interval:      3,
-    max_attempts:        15,
-    on_retry:            Pester::Behaviors::Sleep::Linear
+Retriable.configure do |c|
+  c.contexts[:schema_refresh] = {
+    base_interval: 3,
+    tries: 15,
   }
 
-  c.environments[:s3] = {
-    delay_interval: 30.seconds,
-    max_attempts: 10,
-    on_retry: Pester::Behaviors::Sleep::Constant,
-    reraise_error_classes: [Aws::S3::Errors::NoSuchKey, Aws::S3::Errors::NoSuchBucket]
```

Comment:
hmm what about these reraise classes?

Reply:
there's no blacklist in `Retriable`, so you'd either have to explicitly specify via whitelist what to retry, or i guess possibly wait for this PR: https://github.com/kamui/retriable/pull/55 that would support an arbitrary `Proc` evaluation which could do the whitelisting.

what's the reason for these exceptions not being retried?  is it still even a valid concern?

Reply:
i think these are all unrecoverable errors. ie, retrying isn't going to help if you are missing keys or your bucket config is wrong

Reply:
oh i think this got put in because it was annoying waiting for it to retry when you have the AWS keys misconfigured.  i'll put in a tweak to replicate this behavior.